docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -95,42 +96,106 @@ \section{Team Member Roles}
 \end{itemize}
 
 
-
-\section{Work flow Plan}
-
+\section{Workflow Plan}
+        A general development workflow will be: \
 \begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
```

Comment:
Shall we change "template issues to" issue template? This may more make sense to me. What do you think? Thanks!

Reply:
Hi, when there is a minus sign in front of the lines and the background color is red, it means these lines are removed/changed in this PR